src/FunctionLoader.ts

```diff
@@ -70,6 +71,19 @@ export class FunctionLoader implements IFunctionLoader {
             throw new InternalException(`Function code for '${functionId}' is not loaded and cannot be invoked.`);
         }
     }
+
+    isESModule(filePath: string, packageJson: PackageJson): Boolean {

```

Comment:
nit:

```suggestion
    isESModule(filePath: string, packageJson: PackageJson): boolean {
```

https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html

Reply:
Thanks!